contracts/standard/proxy/ExperimentalProxy.sol

```diff
@@ -40,6 +40,7 @@ contract ExperimentalProxy {
         // Return data
         bytes memory _retData;
 
+        // solium-disable-next-line security/no-inline-assembly
```

Comment:
Not needed. Part of config.

Reply:
I get an error from Solium if I don't add them. Maybe using an old `kleros-scripts`?

Reply:
From running `yarn lint` or from your editor integration?

Reply:
Yeah we are. I'm updating kleros-scripts dependency.